modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -7,42 +7,19 @@
 [id="enabling-fuse-overlayfs-for-all-workspaces"]
 = Enabling fuse-overlayfs for all workspaces
 
+Learn about configuring the workspace's container entrypoint script so that fuse-overlayfs is used for all workspaces using that container.
```

Comment:
```suggestion
Learn how to configure the workspace's container entrypoint script so that you can use fuse-overlayfs for all workspaces using that container.
```